### PR TITLE
Ako/ Poa and Poi notifications

### PR DESCRIPTION
**Pull request by ali-hosseini-fs**
Implementation of Separate notifications for proof of identity and proof of address.

### CODE REVIEW

File: packages/core/src/App/Containers/Layout/header/default-header.jsx [hunk 1]
```diff
@@ -71,6 +73,13 @@ const DefaultHeader = ({
     }, [removeUpdateNotification]);
 
     const onClickDeposit = () => history.push(routes.cashier_deposit);
+
+    const verification_status = populateVerificationStatus(account_status);
+    const { has_poa, has_poi } = verification_status;
+    const onClickUpgrade = () => {
+        // TODO: here we should open POA and POI modal
```
Comment (nikita-deriv) [inline packages/core/src/App/Containers/Layout/header/default-header.jsx:80]: TODO declaration is needed for future functionality or we can remove it?

Comment (ali-hosseini-fs) [reply packages/core/src/App/Containers/Layout/header/default-header.jsx:80]: It is related to modal that Amina has implemented, After we merged that PR I should change this TODO

File: packages/account/src/Sections/Verification/Helpers/verification.js [hunk 2]
```diff
@@ -1,19 +1,25 @@
-export const populateVerificationStatus = account_status => {
-    const { attempts, document, identity, needs_verification } = account_status.authentication;
+const populateVerificationStatus = account_status => {
+    const { attempts, document, identity, needs_verification } = {
+        ...account_status.authentication,
+        attempts: account_status.authentication.attempts || {},
+        needs_verification: account_status.authentication.needs_verification || {},
+    };
```
Comment (amina-deriv) [inline packages/account/src/Sections/Verification/Helpers/verification.js:6]: Can we add  Optional chaining operator(?.) here?

Comment (ali-hosseini-fs) [reply packages/account/src/Sections/Verification/Helpers/verification.js]: done

**Pull request merged**